doc/docs.md

```diff
@@ -2119,6 +2119,42 @@ fn main() {
 }
 ```
 
+To access the result of the function inside a `defer` block the `$res()` expression can be used.
+`$res()` is only used when a single value is returned, while on multi-return the `$res(idx)`
+is parameterized.
+
+```v
```

Comment:
For incomplete snippets, use 3 backspaces, followed by `v ignore`, not just `v`:
```suggestion
```v ignore
```
Even better would be, if there is a small complete example in the docs, so that it would be checked/compiled each time, to prevent regressions.